src/main/java/com/solvd/movie/service/impl/PgMovieServiceImpl.java

```diff
@@ -19,13 +19,13 @@
 @RequiredArgsConstructor
 public class PgMovieServiceImpl implements PgMovieService {
```

Comment:
@hizmailovich 

```java
PgMovies implements Movies
```

WDYT?

Reply:
It can be a good idea, but in my case interfaces `PgMovieService` and `EsMovieService` have different methods and what about repositories